src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -839,6 +838,10 @@ public virtual bool IsDefinedInSourceTree(SyntaxTree tree, TextSpan? definedWith
             return false;
         }
 
+        protected static bool IsDefinedInSourceTree(SyntaxReference syntaxRef, SyntaxTree tree, TextSpan? definedWithinSpan)
```

Comment:
It feels a little unclear from the parameter names that we're testing whether `syntaxRef` refers to a location within `tree`. Would it be reasonable to make this an internal method or extension on `SyntaxReference`?

Reply:
Sure.  added to the list of followups.